tsconfig.json

```diff
@@ -0,0 +1,19 @@
+{
+    "compileOnSave": true,
+    "compilerOptions": {
+        "target": "es6",
```

Comment:
Just set it to `es5` and we can use any ES6 feature, regardless of having to support Node.js 4.

Reply:
The problem is with `new Set()`, which is perfectly used but typescript won't provide a polyfill for es5 during the transpilation, so I would suggest to ponyfill with array and `Array.includes` or an associative Object? 😦

Reply:
Should work if you specify:

```json
"lib": [
	"es2015"
],
```